main_test.go

```diff
@@ -209,6 +209,16 @@ kube_pod_container_status_terminated_reason{namespace="default",pod="pod0",conta
 kube_pod_container_status_terminated_reason{namespace="default",pod="pod0",container="container3",reason="Completed"} 0
 kube_pod_container_status_terminated_reason{namespace="default",pod="pod0",container="container3",reason="Error"} 0
 kube_pod_container_status_terminated_reason{namespace="default",pod="pod0",container="container3",reason="ContainerCannotRun"} 0
+# HELP kube_pod_container_status_terminated_reasons_total Counter with observed container restarts and reason the container was terminated.
```

Comment:
This `kube_pod_container_status_terminated_reasons_total ` can be obtained by  `kube_pod_container_status_terminated_reason` calculations. Do we really need it? @brancz